app/services/disability_claim_service.rb

```diff
@@ -33,6 +33,15 @@ def request_decision(id)
     client.submit_5103_waiver(id).body
   end
 
+  # upload file to s3 and enqueue job to upload to EVSS
+  # TODO (AJM): Unique filename in s3 but preserve original filename when uploading to evss
+  # TODO (AJM): Give user GlobalID for serialization
```

Comment:
Ended up just passing a hash of attrs and recreating the User in the job, ActiveJob cant serialize Time arguments so we're now just sending `va_eauth_issueinstant` as the current time of the API call @carlmjohnson Any ideas around this?